apps/nextjs-website/src/components/atoms/UserInfo/UserInfo.tsx

```diff
@@ -28,20 +57,96 @@ const UserInfo: FC = () => {
       gap={1}
       justifyContent='flex-end'
     >
-      <Hidden smDown>
-        <PersonOutline />
-        <Typography variant='body2' sx={{ fontWeight: 600, fontSize: '14px' }}>
-          {user.attributes.given_name} {user.attributes.family_name}
-        </Typography>
-      </Hidden>
-      <Button
-        size='small'
-        variant='contained'
-        onClick={signOut}
-        sx={{ height: '30px' }}
-      >
-        {logout}
-      </Button>
+      {!user && !isProduction && (
```

Comment:
i see for a split second the login button if i'm logged in when i try to reaload the page
[Screencast from 23-11-2023 18:48:01.webm](https://github.com/pagopa/developer-portal/assets/24475319/f875c816-8aba-4c46-b91c-f5da745b50b9)

i thing u can solve this with `loading` props:
```suggestion
      {!user && !loading && !isProduction && (
```